devGuide.md

```diff
@@ -45,6 +46,7 @@ Each script requires a number of arguments which are outlined below:
 |dburl|The full URL (including credentials) of the CouchDB or Cloudant account used by the feed service.|
 |dbprefix|A prefix to be prepended to the default DB name (ow_kafka_triggers) that will be created by the provider service.|
 |apihost|The hostname or IP address of the core OpenWhisk system that will be used as the hostname for all trigger URLs. In most cases, this will be the same as `edgehost`.|
+|workers|A comma separated list of the IDs of the running instances with each ID of the form `workerX`. e.g. `worker0,worker1`|
```

Comment:
There is an example install script with arguments below. Needs to be updated with workers.